.github/workflows/ci.yml

```diff
@@ -42,11 +42,11 @@ jobs:
         arch:
           - x64
         trixi_test:
+          - 1D
```

Comment:
From Mr. Consistency: Should we name the tests in lowercase `1d`, `2d_parallel` etc. such that they match the filenames they represent? If you think that's over the top, feel free to just close this again ;-)

Reply:
Done in https://github.com/trixi-framework/Trixi.jl/pull/346/commits/e2add030cd4d7ce19e4ea120c3c00e22bd064ceb.